File: src/main/java/ru/citeck/idea/codeinsight/forms/components/PropertyComponent.java
package ru.citeck.idea.codeinsight.forms.components;

import java.util.Set;

public abstract class PropertyComponent extends InputComponent {

    private static final Set<String> ARTIFACT_TYPES = Set.of(
        // TODO: Why alfresco here?
        //AlfrescoModelIndexer.PROPERTY
    );

    @Override
    public Set<String> getSupportedArtifactTypes() {
        return ARTIFACT_TYPES;
    }
}

File: src/main/kotlin/ru/citeck/idea/artifacts/action/codeinsight/forms/components/PropertyComponent.kt
package ru.citeck.idea.artifacts.action.codeinsight.forms.components

abstract class PropertyComponent : InputComponent() {

    companion object {
        private val ARTIFACT_TYPES = setOf<String>(
            // TODO: Why alfresco here?
            //AlfrescoModelIndexer.PROPERTY
        )
    }

    override fun getSupportedArtifactTypes(): Set<String> {
        return ARTIFACT_TYPES
    }
}
